### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by yujincheng08**

### CODE REVIEW

File: core/template_override/README.md [hunk 1]
```diff
@@ -1,47 +1,60 @@
 # LSPosed Framework
 
-[简体中文](https://github.com/ElderDrivers/LSPosed/wiki/%E7%AE%80%E4%BB%8B)
```
Comment (kotori2) [inline core/template_override/README.md:3]: Maybe delete Chinese readme completely

File: core/template_override/README.md [hunk 2]
```diff
@@ -1,47 +1,60 @@
...
+Android 8.1+
 
-## How to use ?
+## Install
 
-To put it simply, just follow these steps:
+1. Install Magisk v21+
+2. Install [Riru](https://github.com/RikkaApps/Riru/releases) v23+ from Magisk repo.
+3. [Download](#download) and install LSPosed in Magisk Manager
+4. Install LSPosed Manager
```
Comment (kotori2) [inline core/template_override/README.md:20]: 1.2.3.4.4.5

**Review by kotori2 — CHANGES_REQUESTED** [review]
I think we should delete this file and let gradle copy it from repo root.

File: core/template_override/README.md (hunk 1 above)
Comment (yujincheng08) [reply core/template_override/README.md:3]: Didn't I?

**Pull request closed without merging**